packages/sourcecred/src/plugins/github/fetchGithubRepo.js

```diff
@@ -313,8 +313,7 @@ export async function postQuery(
   });
   return retryGithubFetch(postBody, token).catch(
     (error: GithubResponseError) => {
-      const type = error.type;
-      switch (type) {
+      switch (error.type) {
```

Comment:
I appreciate the little refactor-as-you-go changes you baked in.

Reply:
heh, flow actually forced me to do this